### PR TITLE
Adding myself to contributor's page

### PR DESCRIPTION
**Pull request by suvansh**
Adding myself (suvansh) to contributor's page

### CODE REVIEW

File: staticfiles/templates/contributors/home.html [hunk 1]
```diff
@@ -208,7 +208,14 @@ <h1 class="page-header">
               <img class="thumbnail img-responsive" src="/static/images/contributors/milstein.png" />
               <span>Milson Munakami</span> 
             </a>
-          </div>		  
+          </div>
+          
+          <div class="col-sm-2 col-xs-4 thumb">
+            <a class="contributor-thumb" href="/contributors/suvansh">
+              <img class="thumbnail img-responsive" src="/static/images/contributors/suvansh.jpg" />
```
Comment (neilthemathguy) [inline staticfiles/templates/contributors/home.html]: suvansh.jpg is missing. Could you upload it?

Comment (suvansh) [inline staticfiles/templates/contributors/home.html]: Should I upload it to the pull request itself?

**Comment by Macbull** [comment]
Apparently you forgot to add suvansh.jpg !! After adding commit and push changes to your forked repo (pull request will be updated automatically)

**Comment by suvansh** [comment]
I'm not clear on where to add suvansh.jpg could you please tell me where to
do so?

**Comment by Macbull** [comment]
1. add suvansh.jpg to /static/images/contributors/
2. commit the change (after staging savannahs.jpg) (git add savannahs.jpg ; git commit -m "adding pic")
3. push the change to your fork (git push origin develop2)

**Comment by neilthemathguy** [comment]
Taking this PR out of the pending list.  Please fix the bugs and raise the new PR or move to the Milestone 2. Thank you.

**Pull request closed without merging**